.github/workflows/check-dist.yml

```diff
@@ -15,3 +15,5 @@ jobs:
   call-check-dist:
     name: Check dist/
     uses: actions/reusable-workflows/.github/workflows/check-dist.yml@main
+with:
```

Comment:
```suggestion
    with:
```

Reply:
Updated the format and removed the conflicts. Please review.